README.rst

```diff
@@ -2994,23 +3091,29 @@ topic model retrieval. This ensures that all fields are retrieved by
 the get method in the same call (unlike in the standard calls where
 the number of fields returned is limited).
 
+If you have not imported our files with Composer and need to include them
+manually, use the lines::
+
+  include '../bigml/bigml.php';
+  include '../bigml/topicmodel.php';
+
+Please note you will still need to use Composer to import the
+third-party stemming library used to create local topic models.
+
 Multi Models
 ------------
 
 Multi Models use a numbers of BigML remote models to build a local version that can be used to generate predictions locally. Predictions are generated combining the outputs of each model::
 
-    require 'vendor/autoload.php';
-
     $multimodel = new MultiModel(array("model/5111xxxxxxxxxxxxxxxxxx12",model/538Xxxxxxxxxxxxxxxxxxx32"));
 
 or::
 
     $multimodel = new MultiModel(array("model/5111xxxxxxxxxxxxxxxxxx12",model/538Xxxxxxxxxxxxxxxxxxx32"), $api);
 
-or set default storage if you have storage unset in api->
-
-    $multimodel = new MultiModel(array("model/5111xxxxxxxxxxxxxxxxxx12",model/538Xxxxxxxxxxxxxxxxxxx32"), null, $storage);
+or set default storage if you have storage unset in::
 
+  $api->$multimodel = new MultiModel(array("model/5111xxxxxxxxxxxxxxxxxx12",model/538Xxxxxxxxxxxxxxxxxxx32"), null, $storage);
 
```

Comment:
again, do you really need to assign this to the $api variable?

Reply:
And here.